x-pack/plugins/observability_solution/dataset_quality/common/translations.ts

```diff
@@ -93,12 +93,9 @@ export const flyoutSummaryText = i18n.translate('xpack.datasetQuality.flyoutSumm
   defaultMessage: 'Summary',
 });
 
-export const overviewDegradedDocsText = i18n.translate(
-  'xpack.datasetQuality.flyout.degradedDocsTitle',
-  {
-    defaultMessage: 'Degraded docs',
-  }
-);
+export const overviewTrendsDocsText = i18n.translate('xpack.datasetQuality.flyout.trendDocsTitle', {
```

Comment:
I think should be plural but trends should be singular hahahah
at the end is the trend of failed docs

Reply:
Maybe @mdbirnstiehl can help us here 😶

Reply:
I would trend (😉) towards using "Document trends" and "Failed document trend" because I believe here we are talking about the singular concept of failed documents in general. If we were talking about a specific set of documents or multiple different types of documents, then we could go with Documents trend.

Reply:
Can we apply the same concept to degraded documents? (e.g. degraded doc trend) ?

Reply:
This is the part in the UI
<img width="1652" alt="image" src="https://github.com/user-attachments/assets/c2bbd21d-5f56-4913-a642-fce2f959a9fb">

> Sorry if this is very confusing.

Indeed 😅

Reply:
Thanks for the visual. Document trends is correct here.